src/cortx-fs.spec-in.cmake

```diff
@@ -89,11 +91,11 @@ rm -rf $RPM_BUILD_ROOT
 %{_cortxfs_bin_dir}/cortxfscli
 %{_bindir}/cortxfscli
 %{_cortxfs_log_dir}
-
 %files devel
 %defattr(-,root,root)
 %{_libdir}/pkgconfig/%{_cortxfs_lib}.pc
 %{_cortxfs_include_dir}/*.h
+%{_cortxfs_include_dir}/REST_doc.html
```

Comment:
Is there any specific reason for adding the HTML file in ` include dir` ? Because we only place the header files in this directory. Instead we can place either in `_cortxfs_bin_dir` or a custom directory holding this HTML doc.